jobs/test-override.Jenkinsfile

```diff
@@ -237,4 +260,36 @@ try {
     currentBuild.description = "[${descPrefix}] ❌"
     currentBuild.result = 'FAILURE'
     throw e
-}}} // cosaPod, timeout, and try finish here
+} finally {
+    // If asked to report results to resultsdb then let's do that now.
+    if (params.REPORT_TO_RESULTSDB) {
+        stage("Report Completion") {
+            def outcome
+            def emoji
+            // treat UNSTABLE as PASSED too; we often have expired snoozed tests
+            // that'll warn and Greenwave/Bodhi treats NEEDS_INSPECTION outcomes
+            // as blocking
+            if (currentBuild.result == 'SUCCESS' || currentBuild.result == 'UNSTABLE') {
+                emoji = "🟢"
+                outcome = 'PASSED'
+            } else {
+                emoji = "🔴"
+                outcome = 'FAILED'
+            }
+            def blueocean_url = "${env.RUN_DISPLAY_URL}"
```

Comment:
Minor/optional: probably not worth a variable?

Reply:
I'll leave the variable in here. It helps my brain understand what `RUN_DISPLAY_URL` is. Otherwise it's not immediately clear to me that it is the blue ocean URL.